Automattic/wp-calypso
# Gutenboarding: Add plans grid working prototype

#### Changes proposed in this Pull Request
* Clicking Plans button (top-right) should open the Plans modal with the following possible actions:
  - select a plan
  - confirm selection (close the modal)
  - cancel selection (close the modal)

#### Testing instructions

* Go to /new in development (`gutenboarding/plans-grid` flag should be enabled).
* Pressing on the top-right Plans button should open the plans grid container.
* Selecting plans and confirm/cancel selection should be possible.

Part of #41787
